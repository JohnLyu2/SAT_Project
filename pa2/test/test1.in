A*B * -A
A*(B * (-A * A) + C) * (C * - C)
A*(B * -A + C)
--B * -(B + C) * A * --C * C * B + (A + (B + B) + B * A) + A
--B * -(B + C) * A * --C * C * B
-((B * (-A * C) * ((C * C * C + C + C) + A * C * C * C)) * (C * B))
(-((B * (-A * C) * ((C * C) + A * C * C * C)) * (C * B)))*B * (C + B + B) * (C + A * C) * (B) * --(B * -(A) + A * B + B + A)
(-((B * (-A * C) * (-C)) * (C * B)))*B * ( B) * (C + A * C) * (B) * --(B * -(A) ) * (C * A)
qwerttyuqe1
(A + A + A) * C + (-(B * C) + (C + B) + B * A) * (B)
