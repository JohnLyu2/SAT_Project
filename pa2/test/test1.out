unsat
unsat
sat
sat
unsat
sat
sat
unsat
Error: invalid input
sat
